src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -1,42 +1,67 @@
-    Props(new CallExecutionActor(backend, command, workflowDescriptor, call, callInputs, lookup))
+
+  sealed trait CallExecutionActorMessage
+  case class Execute(workflowId: WorkflowId,
+                     backend: Backend,
+                     command: String,
+                     workflowDescriptor: WorkflowDescriptor,
+                     call: Call,
+                     callInputs: CallInputs,
+                     lookup: ScopedLookupFunction) extends CallExecutionActorMessage
+
+  def props(callReference: CallReference): Props = Props(new CallExecutionActor(callReference))
 }
 
 /** Actor to manage the execution of a single call. */
-class CallExecutionActor(backend: Backend, command: String, workflowDescriptor: WorkflowDescriptor, call: Call, callInputs: CallInputs, lookup: ScopedLookupFunction) extends Actor with CromwellActor {
+class CallExecutionActor(callReference: CallReference) extends Actor with CromwellActor {
+  import CallExecutionActor._
+
   private val log = Logging(context.system, classOf[CallExecutionActor])
-  val tag = s"CallExecutionActor [UUID(${workflowDescriptor.shortId}):${call.name}]"
-
-  log.info(s"$tag: starting.")
-  backend.executeCommand(command, workflowDescriptor, call, callInputs, lookup) match {
-    case Success(callOutputs) =>
-      log.info(s"$tag: successful execution.")
-      context.parent ! CallActor.ExecutionFinished(callOutputs)
-    case Failure(e) =>
-      log.error(s"$tag: failed execution.")
-      context.parent ! CallActor.ExecutionFailed(e)
+
+  def tag = s"CallExecutionActor [$callReference]"
+
+  private def execution(workflowId: WorkflowId,
+                        backend: Backend,
+                        command: String,
+                        workflowDescriptor: WorkflowDescriptor,
+                        call: Call,
+                        callInputs: CallInputs,
+                        lookup: ScopedLookupFunction) = {
+
+    // Wrap these IDs together so they can be passed around as a unit
+    log.info(s"$tag: starting ${call.name} for workflow ${workflowDescriptor.shortId}")
+
+    val executionResult = backend.executeCommand(command, workflowDescriptor, call, callInputs, lookup, AbortFunctionRegistration(registerAbortFunction(callReference)))
+    executionResult match {
+      case Success(_) => log.info(s"$tag: successful execution.")
+      case Failure(e: TaskAbortedException) => log.info(s"$tag: aborted.")
+      case Failure(e) => log.error(s"$tag: failed execution - ${e.getMessage}")
+    }
+
+    context.parent ! CallActor.ExecutionFinished(call, executionResult)
   }
 
-  /**
-   * The `CallExecutionActor` is not meant to respond to messages, it just starts up, does its work,
-   * and shuts down.
-   */
   override def receive = LoggingReceive {
+    case Execute(workflowId, backend, command, workflowDescription, call, callInputs, lookup) => execution(workflowId, backend, command, workflowDescription, call, callInputs, lookup)
```

Comment:
"Github appears to default to a width of 120" - depends on how you github :)

This is pushing it even for me. But if you dropped the rhs to its own line it's totally fine